### DISCUSSION TITLE
MiraiMC会不会尝试NT框架的QQ机器人？

### DISCUSSION THREAD

**Discussion by Natsusomekeishi** [DreamVoid/MiraiMC#491] [建议 / Ideas]
例如：https://github.com/LagrangeDev/Lagrange.Core

**Comment by DreamVoid** [comment]
问的好，如何把这玩意塞到一个jar包并且不需要第三方程序的帮助即可随意调用？又该如何解决现有mirai体系下的API和事件兼容性呢？
我能想到的解决方法就是写一个新插件，并且我甚至不知道如何解决不同操作系统下的兼容性问题。
